src/utils.js

```diff
@@ -1,4 +1,4 @@
-const fs = require('fs');
+const fs = require('fs-extra');
```

Comment:
Why are we requiring fs-extra ? We have `rmdirSync` in File System module of Node.js itself. Check [here](https://nodejs.org/api/fs.html#fsrmdirsyncpath-options)